packages/@aws-cdk/aws-synthetics/test/integ.canary.ts

```diff
@@ -8,6 +9,8 @@ import * as synthetics from '../lib';
  * Stack verification steps:
  *
  * -- aws synthetics get-canary --name canary-one has a state of 'RUNNING'
```

Comment:
unrelated to your change - should this say `canary-integ`, `canary-one` is the stack name